<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintDialog</class>
 <widget class="QDialog" name="PrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Center</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select the number of portraits you want to print.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="page_num_label">
         <property name="text">
          <string>Pa&amp;ge 1 of 2</string>
         </property>
         <property name="buddy">
          <cstring>prev_page_btn</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="prev_page_btn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/back.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="next_page_btn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/fwd.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Printe&amp;r:</string>
             </property>
             <property name="buddy">
              <cstring>printer_combo</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="printer_combo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Format:</string>
             </property>
             <property name="buddy">
              <cstring>printer_combo</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="media_name_combo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="media_height_lbl">
             <property name="text">
              <string>Media &amp;height</string>
             </property>
             <property name="buddy">
              <cstring>media_height_spin</cstring>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="vert_alignment_lbl">
             <property name="text">
              <string>&amp;Vert alignment</string>
             </property>
             <property name="buddy">
              <cstring>vert_alignment_combo</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="media_unit_combo">
             <item>
              <property name="text">
               <string>points</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>inch</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="horiz_alignment_combo">
             <item>
              <property name="text">
               <string>left</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>center</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>right</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QComboBox" name="vert_alignment_combo">
             <item>
              <property name="text">
               <string>top</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>center</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bottom</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Media &amp;unit:</string>
             </property>
             <property name="buddy">
              <cstring>media_unit_combo</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="media_width_lbl">
             <property name="text">
              <string>Media &amp;width:</string>
             </property>
             <property name="buddy">
              <cstring>media_width_spin</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="media_width_spin">
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Hor&amp;z alignment</string>
             </property>
             <property name="buddy">
              <cstring>horiz_alignment_combo</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="media_height_spin">
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="feed_type_lbl">
             <property name="text">
              <string>Media &amp;type:</string>
             </property>
             <property name="buddy">
              <cstring>feed_type_combo</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="feed_type_combo">
             <item>
              <property name="text">
               <string>sheets</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>roll</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Portraits</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>1&amp;6&quot;x20&quot;:</string>
             </property>
             <property name="buddy">
              <cstring>print_16_20_spin</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="print_16_20_spin"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>1&amp;1&quot;x14&quot;:</string>
             </property>
             <property name="buddy">
              <cstring>print_11_14_spin</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="print_11_14_spin"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>&amp;8&quot;x10&quot;:</string>
             </property>
             <property name="buddy">
              <cstring>print_8_10_spin</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="print_8_10_spin"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>&amp;5&quot;x7&quot;:</string>
             </property>
             <property name="buddy">
              <cstring>print_5_7_spin</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="print_5_7_spin"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="PortraitView" name="graphics_view"/>
       </item>
       <item>
        <widget class="QTextBrowser" name="errors_browser"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="help_btn">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="print_guide_btn">
         <property name="text">
          <string>Sh&amp;ow guide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="print_dialog_btn">
         <property name="text">
          <string>Print &amp;dialog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="print_now_btn">
         <property name="text">
          <string>&amp;Print now!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_btn">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PortraitView</class>
   <extends>QGraphicsView</extends>
   <header>portraitview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>prev_page_btn</tabstop>
  <tabstop>next_page_btn</tabstop>
  <tabstop>printer_combo</tabstop>
  <tabstop>media_unit_combo</tabstop>
  <tabstop>media_width_spin</tabstop>
  <tabstop>media_height_spin</tabstop>
  <tabstop>horiz_alignment_combo</tabstop>
  <tabstop>vert_alignment_combo</tabstop>
  <tabstop>print_16_20_spin</tabstop>
  <tabstop>print_11_14_spin</tabstop>
  <tabstop>print_8_10_spin</tabstop>
  <tabstop>print_5_7_spin</tabstop>
  <tabstop>graphics_view</tabstop>
  <tabstop>errors_browser</tabstop>
  <tabstop>help_btn</tabstop>
  <tabstop>print_guide_btn</tabstop>
  <tabstop>print_dialog_btn</tabstop>
  <tabstop>print_now_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>PrintDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>713</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
